src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.spec.ts

```diff
@@ -459,4 +453,31 @@ describe('LocalGameWrapperComponent', () => {
         // Then the AI metrics are shown
         testUtils.expectElementToExist('#AIMetrics');
     }));
+    describe('takeBack', () => {
+        it('should allow to go back one move', fakeAsync(async() => {
+            // Given a board with a move already done
+            await testUtils.expectMoveSuccess('#click_4', P4Move.FOUR);
+
+            // When taking back
+            spyOn(testUtils.getComponent(), 'updateBoard').and.callThrough();
+            await testUtils.expectInterfaceClickSuccess('#takeBack');
+
+            // Then we should be back on turn 0 and board should have been updated
+            expect(testUtils.getComponent().getTurn()).toBe(0);
+            expect(testUtils.getComponent().updateBoard).toHaveBeenCalledTimes(1);
+        }));
+        it('should cancelMoveAttempt when taking back', fakeAsync(async() => {
+            // Given a board where a move could be ongoing
```

Comment:
tu veux dire quoi par "could be ongoing"?

Reply:
j'crois que "ongoing" dans le sens commencé mais plus achevé aussi clair si pas plus mais ok !
unachieved ce serait mieux ? TPR si t'aime mieux "in construction"